### PR TITLE
update README

### PR DESCRIPTION
**Pull request by wd0517**
This pr introduce how to add hnsw index when creating a table using orm.

### CODE REVIEW

**Review by breezewish — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ TiDB vector supports below distance functions:
 - `CosineDistance`
 - `NegativeInnerProduct`
 
-supports following orm or framework:
+It also supports using hnsw index with l2 or cosine distance to speed up the search, for more details see [Vector Search Indexes in TiDB](https://docs.google.com/document/d/15eAO0xrvEd6_tTxW_zEko4CECwnnSwQg8GGrqK1Caiw)
```
Comment (IANTHEREAL) [inline README.md:22]: We need to replace this google doc by a more formal web doc

File: README.md [hunk 2]
```diff
@@ -211,3 +234,7 @@ tidb_vs.delete(["f8e7dee2-63b6-42f1-8b60-2d46710c1971"])
 # delete with filter
 tidb_vs.delete(["f8e7dee2-63b6-42f1-8b60-2d46710c1971"], filter={"category": "P1"})
 ```
+
+## Examples
```
Comment (IANTHEREAL) [inline README.md:238]: It would be better to introduce examples with a simple but attractive description.

**Review by IANTHEREAL — COMMENTED** [review]
Rest LGTM

File: README.md (hunk 1 above)
Comment (wd0517) [reply README.md:22]: After the public preview, we will create a more formal web document on docs.pingcap.com, and I will update it accordingly.

**Pull request merged**